Tests/SonarQube.MSBuild.Tasks.IntegrationTests/E2ETests/E2EAnalysisTests.cs

```diff
@@ -630,7 +630,7 @@ private void CheckAnalysisFileList(ProjectDescriptor expected, string projectOut
                 {
                     foreach(string unanalysedFile in expected.FilesNotToAnalyse)
                     {
-                        CollectionAssert.DoesNotContain(expectedFiles, unanalysedFile, "Not expecting file to be included for analysis: {0}", unanalysedFile);
```

Comment:
Error in previous checkin: should be checking that the file that should not be analysed is not in the list of actual files.